.changeset/curvy-apes-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopify/shopify-app-remix": patch
+---
+
+Fix back stack for embedded apps
```

Comment:
Mind if we just add a little more colour here? This is the main way for folks to catch up on what we're doing, so I like to be very explicit in the changelog.
```suggestion
Fix a bug that prevents using the browser's back stack for embedded apps
```